settings-templates/harmonium-settings.scss

```diff
@@ -14,26 +14,27 @@ $breakpoints: (
   small: (min-width: $small),
   small-only: ((min-width: $small) and (max-width: $medium - 1)),
   medium: (min-width: $medium),
-  medium-down: ((max-width: $medium - 1)),
+  medium-down: ((max-width: $medium)),
   medium-only: ((min-width: $medium) and (max-width: $large - 1)),
   large: (min-width: $large),
-  large-down: ((max-width: $large - 1)),
+  large-down: ((max-width: $large)),
   large-only: ((min-width: $large) and (max-width: $xlarge - 1)),
   xlarge: (min-width: $xlarge),
-  xlarge-down: ((max-width: $xlarge - 1)),
+  xlarge-down: ((max-width: $xlarge)),
   xlarge-only: ((min-width: $xlarge) and (max-width: $xxlarge - 1)),
   xxlarge: (min-width: $xxlarge),
   global-width: (min-width: $global-width),
   nav: (min-width: $nav), // Desktop nav
-  nav-only: ((min-width: $small) and (max-width: $nav - 1)), // Mobile nav
+  nav-only: ((min-width: $small) and (max-width: $nav - 1)) // Mobile nav
 );
 
 // IMPORT FONTS HERE
 
 // Global Fonts
 $global-monospace-font: 'Montserrat', sans-serif;
 $global-serif-font: 'Georgia', serif;
-$global-sans-serif-font: 'Roboto', 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande', sans-serif;
+$global-sans-serif-font: 'Roboto', 'Helvetica Neue', Helvetica, Arial,
```

Comment:
looks like something got changed here so lets get this changed back too

Reply:
Not sure why, but my editor broke off that line into a newline. So the fonts are all still there but I changed it back to all be on the same line.